README.rst

```diff
@@ -3,12 +3,11 @@ nbgitpuller
 ===========
 
 ``nbgitpuller`` lets you distribute content in a git repository to your
-students by having them click a simple link. :ref:`Automatic, opinioned
-conflict resolution <topic/automatic-merging>` ensures that your students are
+students by having them click a simple link. `Automatic merging <https://jupyterhub.github.io/nbgitpuller/topic/automatic-merging.html>`_ ensures that your students are
 never exposed to ``git`` directly. It is primarily used with a JupyterHub,
 but can also work on students' local computers.
 
-.. image:: docs/_static/nbpuller.gif
+.. image:: https://raw.githubusercontent.com/jupyterhub/nbgitpuller/v0.8.0/docs/_static/nbpuller.gif
```

Comment:
This is needed for the image to appear in the pypi readme. I've used the last tag instead of `master` so that if the docs are rearranged in future old pypi readme's won't break.